extensions/amp-facebook/0.1/test/test-amp-facebook.js

```diff
@@ -20,7 +20,9 @@ import {adopt} from '../../../../src/runtime';
 
 adopt(window);
 
-describe('amp-facebook', () => {
+describe('amp-facebook', function() {
```

Comment:
yeah. agreed. I think overall third party rendering seems flakey so maybe worth visiting to mock out of the tests.